### DISCUSSION TITLE
How to stop jupyter from printing survey notification?

### DISCUSSION THREAD

**Discussion by HildaJ** [microsoft/vscode-jupyter#14235] [Questions and Answers]
Same issue has been reported earlier though (See link below), the latest jupyter extension still pops up survey notification every time I run vscode. Is there an option to disable showing notifications?

Link: https://github.com/microsoft/vscode-jupyter/issues/1700

**Comment by DonJayamanne** [comment]
Sorry you are running into this
 Please can you share a screenshot of the prompt, 
The issue you referenced has prompted from python extension as well

**Reply by HildaJ** [reply]
Hi Don,

Adding to the comments by @surajx in the reference link, when I execute any cell in a Jupyter notebook everyday, then the survey notification show up. Different from his working environment, I run R rather than python codes.  

However, this prompt doesn't show up anymore for 3 days after I asked this question, I guess this prompt only exists for who newly installed Jupyter extension. That explains why @marchmallow said the pop-ups stop showing up later.

Thanks
